src/templates/invoice/Invoice.tsx

```diff
@@ -115,7 +115,7 @@ export const InvoiceTemplate: FunctionComponent<TemplateProps<Invoice>> = ({ doc
         </div>
       </div>
       <div className="flex flex-wrap text-center mb-4 md:mb-0" />
-      <table className="table-auto w-full">
+      <table className="table-auto w-full table-responsive">
         <thead>
```

Comment:
`.table-responsive` does nth here, rmb we no longer use bootstrap? what i meant was to follow how bootstrap deal with tables in small screens, what they have is a wrapper div with `.table-responsive`. let's look deeper in what styles is defined in that: https://github.com/twbs/bootstrap/blob/main/scss/_tables.scss#L147

u see `overflow-x` and `-webkit-overflow-scrolling` is defined there, so what u can do is copy that implementation details onto your case now. check if it resolves your mobile overflowing issue